include/rtc/configuration.hpp

```diff
@@ -35,8 +35,9 @@ struct IceServer {
 	IceServer(const string &host_);
 	IceServer(const string &hostname_, uint16_t port_);
 	IceServer(const string &hostname_, const string &service_);
-	IceServer(const string &hostname_, const string &service_, Type type_, string username_,
-	          string password_, RelayType relayType_);
+	IceServer(const string &hostname_, const string &service_, string username_, string password_);
+	IceServer(const string &hostname_, const string &service_, string username_, string password_,
```

Comment:
Use this Constructor for TURN Server

Reply:
I might suggest something simpler, why not merge the two types in one, like:
```
enum class Type { Stun, TurnUdp, TurnTcp, TurnTls };
```

Reply:
I think we should keep ```RelayType``` as it is. 
It must be same as NiceRelayType.